### ISSUE TITLE
[FEATURE] Adding an optional 'date' & 'hours' field to the 'zeit export' command.

### ISSUE THREAD

**Issue by MaximilianSoerenPollak** [mrusme/zeit#32]
Hey, just something I've been implementing in my personal varation of this as I find it quite usefull is to have two additional fields in the 'export' JSON that comes out of `zeit export` . 

- Date (showing 'DD-MM-YYYY') 
- Hours (Hours,  with decimal precision, that this task has been tracked) 

Now I think since maybe not everyone wants these it would be best to make them 'optional' so that you have to enable them. 

Some questions for that, that I would have then would be. 
 
 - What should the 'shortcuts' for the flags be? (-h and -d)  ? 

Let me know if this is something that might be interesting, and I can make a PR for it in time. 
And also, any additional requierements or things it should do.

Best

**Comment by mrusme** [comment]
Sounds good to me! In order to not break people's scripts, making the output optional is a good idea. 

`-h` and `-d` won't conflict with anything else, however, I try to limit short flags for things that are available across multiple commands, like `-p` and `-t`. Since I don't see any other command that you might want to use these flags in, I would prefer them to be long flags (`--date`, `--hours-decimal`?) without a short variant.

What do you think?

**Comment by MaximilianSoerenPollak** [comment]
Quick question here, I just ran the 'test suite' and the error 
```
# github.com/mrusme/zeit/z
z/statsCmd.go:55:29: undefined: group
FAIL
```
Came up, and I checked, group does seem undefined, and I have not deleted it. Do you want me to fix this too in the same PR?

**Comment by MaximilianSoerenPollak** [comment]
One question that I do have. 

Should I also adapt the 'tyme' export? I'm not sure how that app/program functions and I don´t know if it can handle extra fields etc. 

Let me know :)

**Comment by mrusme** [comment]
Nope, I don't think the `tyme` export needs adaptation. They have a fixed format that they can export/import, and that's what's implemented rn. I don't use `tyme` anymore, but I didn't remove the implementation in case anyone still is.

Let me get back to the other things in your PR.
